recipe/patches/0005-move-zoneinfo_file-function-into-__gnu_cxx-namespace.patch

```diff
@@ -0,0 +1,120 @@
+From 604f16b22f754741c8ac7a9b7dc13a97d3e0a15f Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Mon, 1 Jul 2024 09:34:14 +1100
+Subject: [PATCH 5/6] move zoneinfo_file function into __gnu_cxx namespace
```

Comment:
Let's not move functions around like this. I don't think we should be changing the namespaces in libstdc++. If I read the code correctly, this is for testing?

Reply:
There are more details in the commit message of https://github.com/conda-forge/ctng-compilers-feedstock/pull/142/commits/9ffd5185d5e5f89450f846f06b712b59de11e0de; without moving this function, we have no way to test our implementation of `zoneinfo_dir_override` (except implicitly through loading a timezone), because declaring `extern ... zoneinfo_dir_override` actually changes the effect that the function has, in the sense that the "root" we're using for the relative paths is not `libstdc++` anymore, but the TU where we're calling `zoneinfo_dir_override`.